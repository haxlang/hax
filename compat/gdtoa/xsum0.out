README	fbd27a99	17749
arithchk.c	4bb1e54	4064
dmisc.c	e0d5f990	4773
dtoa.c	f59614cc	18235
g_Qfmt.c	ce0dd69	2945
g_Qfmt_p.c	fb5c0391	3352
g__fmt.c	1ce54774	4638
g_ddfmt.c	cb78a82	4286
g_ddfmt_p.c	e8060288	4890
g_dfmt.c	ebdfd6b1	2648
g_dfmt_p.c	32b3862	3061
g_ffmt.c	f037fcc2	2529
g_ffmt_p.c	7edb123	2844
g_xLfmt.c	eee688e	2792
g_xLfmt_p.c	e4f82595	3145
g_xfmt.c	7a908f5	2896
g_xfmt_p.c	1907a7a5	3349
gdtoa.c	15f30219	17420
gdtoa.h	eaadc6a8	5359
gdtoa_fltrnds.h	1aaf5112	421
gdtoaimp.h	131b5d0d	21889
gethex.c	e16371ae	7360
gmisc.c	1859d016	2084
hd_init.c	f561fe9c	2525
hexnan.c	e337dab	3636
makefile	1d70a257	3551
misc.c	1d181bfd	16141
printf.c	15c97ab	158
printf.c0	f8fda84b	29819
qnan.c	f8e9e8cf	3547
smisc.c	1d9fbad	3727
stdio1.h	b6acbbe	3058
strtoIQ.c	f3272d6f	2013
strtoId.c	fabd9105	2005
strtoIdd.c	11248808	2193
strtoIf.c	16750d29	1949
strtoIg.c	a0bcf2	3720
strtoIx.c	e7af77b4	2034
strtoIxL.c	a83e84a	2005
strtod.c	6a63adb	23026
strtodI.c	1e8708d3	4018
strtodg.c	189daf00	21924
strtodnrp.c	1489d0a1	2552
strtof.c	efc8e192	2200
strtopQ.c	e8127161	2729
strtopd.c	e17050bd	1765
strtopdd.c	e7851577	4668
strtopf.c	e8c661a1	2169
strtopx.c	f4cd73b2	2811
strtopxL.c	1fe9ea0f	2547
strtorQ.c	125fffcb	2968
strtord.c	f943330	2558
strtordd.c	9cf5b6b	5210
strtorf.c	1d26a04f	2459
strtorx.c	4055fd4	3058
strtorxL.c	fa548382	2794
sum.c	2bcf893	2518
ulp.c	edb56c0b	1866
