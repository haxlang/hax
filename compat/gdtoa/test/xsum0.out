README	efba0d5d	3412
Q.ou0	e5903210	28961
Q.ou1	c0a0cec	39848
Qtest.c	39c7381	5823
d.out	fbd2cc70	28614
dI.out	12d50f0d	4517
dIsi.out	1ecb6a38	4498
dItest.c	375fe4e	2372
dd.out	feae6ba5	41333
ddsi.out	1dba4cfd	10661
ddtest.c	f664a218	5631
dt.c	a240b56	7074
dtest.c	fa7324f5	4738
dtst.out	71e63d3	24237
f.out	ea035e24	22461
ftest.c	fe6ae5f0	4736
getround.c	ee95ed1	2502
makefile	f714a641	5634
pfLqtestnos	ffb9723	99
pftest.c	ea314d7f	3452
pftestQ.out	198434fd	830
pftestx.out	1ccea5dd	788
pftestLq.out	1691dbfc	845
pftestnos	ecbc9be6	101
rtestnos	f94bcdf6	336
strtoIdSI.c	7bfb88b	49
strtoIddSI.c	72e8852	50
strtodISI.c	ed08b740	49
strtodt.c	f1aa53af	3374
strtopddSI.c	13e7138d	50
strtorddSI.c	f7e4b1d5	50
testnos	fb521644	498
testnos1	7e16229	294
testnos3	f5ae7ef3	14403
x.ou0	1426698f	25576
x.ou1	e196c6c1	34824
xL.ou0	f85f5eb6	26576
xL.ou1	1550d69c	37429
xLtest.c	f1964154	5556
xQtest.c	efbe29be	1912
xtest.c	1b76bf93	5541
